src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,32 @@
+package Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.*;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+public class HomePage extends BasePage {
+
+@FindBy (css = "#playlists > ul > li:nth-child(4) > a")
+WebElement firstPlaylist;
+@FindBy (css = "input[name='name']")
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```